akka-docs/rst/java/lambda-persistence.rst

```diff
@@ -422,22 +422,19 @@ this will send the message to the destination actor. When recovering, messages w
 Once recovery has completed, if there are outstanding messages that have not been confirmed (during the message replay), 
 the persistent actor will resend these before sending any other messages.
 
-Deliver also requires a function to pass the ``deliveryId`` into the message. A ``deliveryId`` is required to acknowledge 
-receipt of a message, and is also used in playback, when the actor is recovering so that messages received can be correctly acknowledged. 
-A function can be created to map your own ``messageId`` to ``deliveryId``, which may come from your own domain model. 
-This function must keep track of which ``messageId`` have been acknowledged.
-Alternatively, the Persistence module provides a default sequence number implementation which can also be used as the ``deliveryId`` 
-for messages. The default sequence increases monotonically, without gaps.
+Deliver requires a ``deliveryIdToMessage`` function to pass the ``deliveryId`` into the message so that correlation 
+between ``deliver`` and ``confirmDelivery`` is possible. The ``deliveryId`` must do the round trip. Upon receipt 
+of the message destination actor will send the same``deliveryId`` wrapped in a confirmation message back to the sender. 
```

Comment:
Final remark then it's LGTM and ready to be merged!

Missing punctuation in this sentence, let's clarify: `Upon receipt of the message, the destination actor will send `. Otherwise it could be read as "message destination".